packages/@aws-cdk/aws-ec2/README.md

```diff
@@ -577,6 +577,33 @@ const mySecurityGroupWithoutInlineRules = new ec2.SecurityGroup(this, 'SecurityG
 mySecurityGroupWithoutInlineRules.addIngressRule(ec2.Peer.anyIpv4(), ec2.Port.tcp(22), 'allow ssh access from the world');
 ```
 
+### Importing an existing security group
+
+If you know the ID and the configuration of the security group to import, you can use `SecurityGroup.fromSecurityGroupId`:
+
+```ts
+const sg = ec2.SecurityGroup.fromSecurityGroupId(this, 'SecurityGroupImport', 'security-group-import', {
+  allowAllOutbound: true,
+});
+```
+
+Alternatively, use method `SecurityGroup.fromLookupAttributes` to import security groups if you do not know the ID or the configuration details. You can lookup a security group by `securityGroupId` or by `securityGroupName`.
+
+The result of the `SecurityGroup.fromLookupAttributes` operation will be written to a file called `cdk.context.json`. You must commit this file to source control so that the lookup values are available in non-privileged environments such as CI build steps, and to ensure your template builds are repeatable.
+
+```ts fixture=with-vpc
+const sg = ec2.SecurityGroup.fromLookupAttributes(
```

Comment:
What do you think about dedicated `fromLookupById` and `fromLookupByName` methods? They could both still use a (private) `fromLookupAttributes` method, but it may make the API intent a bit more clear.

Reply:
I tried to create an API similar to [VpcLookupOptions](https://docs.aws.amazon.com/cdk/api/latest/docs/@aws-cdk_aws-ec2.VpcLookupOptions.html). Now I compared it to `fromLookupById` and `fromLookupByName` again. For me, this seems to be a better option because the necessary parameters are defined by the API. I've implemented the following changes:
- Deprecated method `fromLookup`, will be replaced by `fromLookupById` (same parameters).
- New method `fromLookupById` with parameter `securityGroupId`. Parameter `vpc` not defined because the security group id is sufficient to clearly identify the security group.
- New method `fromLookupByName` with parameters `securityGroupName` and `vpc`.  `vpc` is defined as required parameter because it avoids problems if the same security group name is used multiple times per account and region.   A security group is used in context of a VPC. Therefore, the VPC should be available in the CDK code. If it should be defined as optional later, we could changed it easily (no breaking change).